client/landing/jetpack-cloud/components/log-item/docs/example.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+
+import React, { PureComponent, ReactNode } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import LogItem from '../';
+import CardHeading from 'components/card-heading';
+
+export default class LogItemExample extends PureComponent {
+	static displayName = 'LogItemExample';
```

Comment:
Did you mean to make this `LogItem`?

Reply:
Resolved in 16e29d65ee8ef054a490d8d8bfa19914bf5b52d7